### PR TITLE
Added editable messages

### PR DESCRIPTION
**Pull request by montlikadani**
**If it is accepted then I will add it to the other message files.**

### CODE REVIEW

File: Essentials/src/messages.properties [hunk 1]
```diff
@@ -475,8 +477,8 @@ skullChanged=\u00a76Skull changed to \u00a7c{0}\u00a76.
 slimeMalformedSize=\u00a74Malformed size.
 socialSpy=\u00a76SocialSpy for \u00a7c{0}\u00a76\: \u00a7c{1}
 socialSpyMsgFormat=\u00a76[\u00a7c{0}\u00a76 -> \u00a7c{1}\u00a76] \u00a77{2}
-socialSpyMutedPrefix=\u00a7f[\u00a76SS\u00a7f] \u00a77(muted) \u00a7r
-socialSpyPrefix=\u00a7f[\u00a76SS\u00a7f] \u00a7r
+socialSpyMutedPrefix=\u00a7f[\u00a76SS\u00a7f] \u00a77(muted) \u00a7r{0}: {1}
+socialSpyPrefix=\u00a7f[\u00a76SS\u00a7f] \u00a7r{0}: {1}
```
Comment (mdcfe) [inline Essentials/src/messages.properties]: I'm not sure about this, the `Prefix` is supposed to go at the start. Also, if this is changed then suddenly everyone's custom messages will break.

Maybe rename these to `socialSpyMuted` and `socialSpy`?

**Review by mdcfe — CHANGES_REQUESTED** [review]

File: Essentials/src/messages.properties (hunk 1 above)
Comment (montlikadani) [reply Essentials/src/messages.properties]: Oh, right... I did not take it. This is not good, I'll restore it.

**Review by mdcfe — DISMISSED** [review]

File: Essentials/src/messages.properties [hunk 2]
```diff
@@ -253,6 +255,7 @@ leatherSyntax=\u00a76Leather color syntax\: color\:<red>,<green>,<blue> eg\: col
 lightningSmited=\u00a76Thou hast been smitten\!
 lightningUse=\u00a76Smiting\u00a7c {0}
 listAfkTag=\u00a77[AFK]\u00a7r
+listAfkHiddenTagFormat=\u00a7f{0}
```
Comment (mdcfe) [inline Essentials/src/messages.properties]: In `PlayerList.java`, the `\u00a7f` is appended *after* the display name (`{0}`), but here, it's before the display name. Is this intentional?

**Review by mdcfe — CHANGES_REQUESTED** [review]

File: Essentials/src/messages.properties (hunk 2 above)
Comment (montlikadani) [reply Essentials/src/messages.properties]: Oh, I thought changing the player's display name in chat. I'm just looking at the method. :(

**Review by mdcfe — DISMISSED** [review]
Everything looks good now.

File: Essentials/src/messages_hu.properties [hunk 3]
```diff
@@ -207,9 +207,8 @@ inventoryClearingAllItems=\u00a76Az \u00f6sszes dolog t\u00f6r\u00f6lve lett {0}
 inventoryClearingAllStack=\u00a76T\u00f6r\u00f6lve minden \u00a7c{0} {1}\u00a76 eszk\u00f6zt\u00e1r\u00e1b\u00f3l.
 inventoryClearingFromAll=\u00a76\u00d6sszes j\u00e1t\u00e9kos eszk\u00f6zt\u00e1r\u00e1nak \u00fcr\u00edt\u00e9se...
 inventoryClearingStack=\u00a76Elt\u00e1vol\u00edtva \u00a7c{0} \u00a76db \u00a7c{1} \u00a76{2}\u00a76 eszk\u00f6zt\u00e1r\u00e1b\u00f3l.
-is=is
-isIpBanned=\u00a76IP \u00a7c{0} \u00a76m\u00e1r ki van bannolva.
-internalError=\u00a7cBels\u00f5 hiba tört\u00e9nt a parancs v\u00e9grehajt\u00e1sa k\u00f6zben.
```
Comment (mdcfe) [inline Essentials/src/messages_hu.properties]: This merge accidentally removed `internalError` from `messages_hu.properties`. It should be readded.

**Review by mdcfe — CHANGES_REQUESTED** [review]

**Review by mdcfe — DISMISSED** [review]

**Review by mdcfe — APPROVED** [review]

**Comment by mdcfe** [comment]
Thanks for the PR!

**Pull request merged**